mcc/odin/odin/mock_rds_client_test.go

```diff
@@ -16,6 +16,30 @@ type mockRDSClient struct {
 	dbSnapshots          []*rds.DBSnapshot
 }
 
+// AddSnapshots add a list of snapshots to the mock, both in the full
+// list and in the per instance map.
+func (m *mockRDSClient) AddSnapshots(
```

Comment:
`*` means it is a pointer to the mockRDSClient, pretty much like in C. About the descriptivity... well, probably we could use something else, but this is just a reference to the object the method is linked to.
So `AddSnapshots()` is a method to `mockRDSClient`, the way to establish this relationship, is making this. In some way, this is like `self` in Ruby, or `this` in Python. this is just the reference. Since Go is more like C, when it comes to reference and value passing, when the method changes the value/state of the object, it needs to use a pointer.